config/testgrids/config.yaml

```diff
@@ -1645,6 +1645,9 @@ dashboards:
   - name: verify-yaml
     test_group_name: pull-test-infra-yamllint
     base_options: width=10
+  - name: verify-manifests
```

Comment:
As part of or after this PR, the `presubmits-test-infra` dashboard should go in `testgrids/kubernetes/sig-testing` or `testgrids/kubernetes/test-infra`